plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaSplitManager.java

```diff
@@ -81,8 +81,8 @@ public ConnectorSplitSource getSplits(
             partitionEndOffsets = kafkaFilteringResult.getPartitionEndOffsets();
 
             ImmutableList.Builder<KafkaSplit> splits = ImmutableList.builder();
-            Optional<String> keyDataSchemaContents = contentSchemaReader.readKeyContentSchema(kafkaTableHandle);
-            Optional<String> messageDataSchemaContents = contentSchemaReader.readValueContentSchema(kafkaTableHandle);
+            Optional<String> keyDataSchemaContents = contentSchemaProvider.readKeyContentSchema(kafkaTableHandle);
+            Optional<String> messageDataSchemaContents = contentSchemaProvider.readValueContentSchema(kafkaTableHandle);
```

Comment:
Should methods be renamed to `.get..` ?
Can methods be renamed to ommit the `ContentSchema` suffix?

Reply:
Yep, added as a separate commit. Thanks!